sdk/core/ci.yml

```diff
@@ -21,6 +21,7 @@ trigger:
   paths:
     include:
     - sdk/core/
+    - eng/
```

Comment:
Shouldn't this be added to both PR and Trigger sections.

Reply:
Yes it would be good to have it in both.

Reply:
Added.